src/main/java/step2/domain/LottoIssuer.java

```diff
@@ -0,0 +1,32 @@
+package step2.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class LottoIssuer {
+    private static final int LOTTO_PRICE = 1000;
+    private static final List<Integer> LOTTO_NUM_STORE = new ArrayList<>();
+
+    static {
+        for(int i = 1; i <= 45; i++){
+            LOTTO_NUM_STORE.add(i);
+        }
+    }
```

Comment:
재사용을 위한 로또 번호 숫자 캐싱 👍
정수타입으로 가지지 않고  LottoNumber 타입으로 변환해서 가지고 있는다면 
불필요하게 LottoNumber 를 생성할 일이 줄어 들고 재사용성이 더 좋아질 것 같은데 어떻게 생각하시나요 :) ?

```suggestion
    private static final List<LottoNumber> LOTTO_NUMBER_STORE = new ArrayList<>();

    static {
        for (int i = 1; i <= 45; i++) {
            LOTTO_NUMBER_STORE .add(new LottoNumber(i));
        }
    }
```

Reply:
이전에 드렸던 해당 피드백을 한번 고민해주시면 좋을 것 같습니다 :)
https://github.com/next-step/java-lotto/pull/843/files#r487521113